packetbeat/beater/packetbeat.go

```diff
@@ -103,15 +98,7 @@ func (pb *Packetbeat) init(b *beat.Beat) error {
 		return err
 	}
 
-	queueSize := defaultQueueSize
-	if b.Config.Shipper.QueueSize != nil {
-		queueSize = *b.Config.Shipper.QueueSize
-	}
-	bulkQueueSize := defaultBulkQueueSize
-	if b.Config.Shipper.BulkQueueSize != nil {
-		bulkQueueSize = *b.Config.Shipper.BulkQueueSize
-	}
-	pb.Pub, err = publish.NewPublisher(b.Publisher, queueSize, bulkQueueSize)
+	pb.Pub, err = publish.NewPublisher(b.Publisher, *b.Config.Shipper.QueueSize, *b.Config.Shipper.BulkQueueSize)
```

Comment:
This currently does not work as the publisher configs are only loaded after the packetbeat init function.